### DISCUSSION TITLE
is poetry able to use the default pyproject.toml file from pypa sample python package?

### DISCUSSION THREAD

**Discussion by vmgustavo** [python-poetry/poetry#8484] [Q&A]
Poetry has its own toml section to read from that specify the metadata to build a python package. However there is a default toml file the works with pip but when I try to run it with poetry it fails to execute because:

```
[tool.poetry] section not found in pyproject.toml
```

is it somehow possible to make both pip and poetry work with the same pyproject.toml without redundancy?

**Comment by Hellseher** [comment]
pyproject.toml is standardised for a long time, it would be reasonable to relay on expected sections instead of mirroring it in custom `tool.poetry` section.

**References**
- [PEP 621 – Storing project metadata in pyproject.toml](https://peps.python.org/pep-0621/)
- [Packaging pyproject.toml specification](https://packaging.python.org/en/latest/specifications/pyproject-toml/)
- [PIP pyproject.toml](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/)

**Issues**
- https://github.com/python-poetry/poetry/issues/3332

**Reply by dimbleby** [reply]
cool, please submit a pull request

**Comment by Hellseher** [comment]
@dimbleby For my current purpose I've failed over to use just `pip` as it's completely
compatible with PEP621 and most of the time comes with Python core.
